src/normalization/normalizeTenant.ts

```diff
@@ -1,13 +1,21 @@
+import ogs from 'open-graph-scraper';
 import { Tenant, TenantAttributes } from '../models/TenantModel';
 import { UserAttributes } from '../models/UserModel';
 import { LocationAttributes } from '../models/LocationModel';
+import reportError from '../common/reportError';
 
 interface NormalizedTenant extends Omit<TenantAttributes, 'userId'|'locationId'> {
   user?: Omit<UserAttributes, 'createdAt'>,
   location?: LocationAttributes,
+  image?: {
+    url: string;
+    width: string;
+    height: string;
+    type: string;
+  } | null
```

Comment:
Let's make the separate type for the image and I suppose all the variables should be required.